components/fraudlabs_pro/actions/screen-order/screen-order.mjs

```diff
@@ -0,0 +1,272 @@
+      optional: true,
+    },
+    avs_result: {
+      type: "string",
+      label: "avs_result",
+      description: "*(optional)* The single character AVS result returned by the credit card processor. Please refer to [AVS & CVV2 Response Codes](https://www.fraudlabspro.com/developer/reference/avs-and-cvv2-response-codes) for details.",
+      optional: true,
+    },
+    cvv_result: {
+      type: "string",
+      label: "cvv_result",
+      description: "*(optional)* The single character CVV2 result returned by the credit card processor. Please refer to [AVS & CVV2 Response Codes](https://www.fraudlabspro.com/developer/reference/avs-and-cvv2-response-codes) for details. NOTE: This is not for the input of the actual CVV code from the back of the credit card.",
+      optional: true,
+    },
+    user_order_id: {
+      type: "string",
+      label: "user_order_id",
+      description: "*(optional)* Merchant identifier to uniquely identify a transaction. It supports maximum of 15 characters user order id input.",
+      optional: true,
+    },
+    user_order_memo: {
+      type: "string",
+      label: "user_order_memo",
+      description: "*(optional)* Merchant description of an order transaction. It supports maximum of 200 characters.",
+      optional: true,
+    },
+    amount: {
+      type: "string",
+      label: "amount",
+      description: "*(optional)* Amount of the transaction.",
+      optional: true,
+    },
+    quantity: {
+      type: "string",
+      label: "quantity",
+      description: "*(optional)* Total quantity of the transaction.",
+      optional: true,
+    },
+    currency: {
+      type: "string",
+      label: "currency",
+      description: "*(optional)* Currency code used in the transaction. It requires the input of ISO-4217 (3 characters) currency code, e.g. `USD` for US Dollar. Please refer to [Currency Codes](https://www.fraudlabspro.com/developer/reference/currency-codes) for complete list.",
+      optional: true,
+    },
+    department: {
+      type: "string",
+      label: "department",
+      description: "*(optional)* Merchant identifier to uniquely identify a product or service department.",
+      optional: true,
+    },
+    payment_mode: {
+      type: "string",
+      label: "payment_mode",
+      description: "*(optional)* Payment mode of transaction. Valid values: `creditcard` | `affirm` | `paypal` | `googlecheckout` | `bitcoin` | `cod` | `moneyorder` | `wired` | `bankdeposit` | `elviauthorized` | `paymitco` | `cybersource` | `sezzle` | `viabill` | `amazonpay` | `pmnts_gateway` | `giftcard` | `ewayrapid` | `others`.",
+      optional: true,
+    },
+    flp_checksum: {
+      type: "string",
+      label: "flp_checksum",
+      description: "*(optional)* Checksum for the device validation. Please visit [Agent Javascript](https://www.fraudlabspro.com/developer/javascript) to learn about the use of this parameter.",
+      optional: true,
+    },
+  },
+  async run() {
```

Comment:
Please add the dollar sign object. See the docs [here](https://pipedream.com/docs/components/guidelines/#use-summary-to-summarize-what-happened)
```suggestion
  async run({ $ }) {
```